CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ Adding a new version? You'll need three changes:
   [#449](https://github.com/Kong/kubernetes-configuration/pull/449)
   [#452](https://github.com/Kong/kubernetes-configuration/pull/452)
 
+### Added
+
+- Added sticky sessions support for `KongUpstreamPolicy`.
```

Comment:
I'm confused on the naming of this feature: is this drain support or sticky sessions? I believe we should be consistent on the naming

Reply:
👍 

As discussed on sync: let's add a mention how this relates to drain support. These 2 features are unrelated but when both are enabled then behavior changes. Ideally we'd make a note of that here. WDYT?